bundles/binding/org.openhab.binding.zwave/database/aeon/zw120-a.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+Aeotec ZW120 Door/Window Sensor Gen5, base on Engineering Spec 08/01/2016 
+from https://github.com/OpenZWave/open-zwave/blob/master/config/aeotec/zw120.xml
+-->
+<Product>
+	<Model>ZW120-A</Model>
```

Comment:
cdjackson: I've added it to your database.  Hopefully, this is okay.  Thanks again for all your hard work.